src/NUnitFramework/tests/Internal/RandomGeneratorTests.cs

```diff
@@ -134,5 +134,46 @@ public static void RandomFloatsAreUnique()
             UniqueValues.Check(values, 8); // Heuristic
         }
         #endregion
+    
+        #region String & Guids
+        
+        [TestCase]
```

Comment:
TestCase without arguments? I'm a bit surprised that it works. Anyway, this test seems to be superfluous compared to the next one.